windows/jitsi-meet/app.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+import http from '../../modules/httpcontrol';
+import handleCommand from '../../modules/commandhandler';
+import config from '../../config.js';
+import App from '../../components';
+
+const listeningPort = config.PORT;
```

Comment:
I'd rename the constant to `LISTENING_PORT`. It looks clearer to me that this is a constant.